.coveragerc

```diff
@@ -2,6 +2,7 @@
 omit = analyticsdataserver/settings*
        *wsgi.py
        analytics_data_api/management/commands/generate_fake_course_data.py
+       *test*
```

Comment:
Any particular reason for omitting the tests from coverage data?  Ned has [opinions on this](https://nedbatchelder.com/blog/201908/dont_omit_tests_from_coverage.html).

Reply:
Good post, doing the change